.travis.yml

```diff
@@ -2,7 +2,6 @@ language: php
 
 php:
   - 7.0
```

Comment:
@dereuromark I think the idea was to trigger the coverage report ASAP if I remember well